store.php

```diff
@@ -1,2 +1,17 @@
 <?php
 
+// read the file
+require_once('./models/task.php');
+
+//receive file
+$title = $_POST['title'];
+$contents = $_POST['contents'];
+$timestamp = time("Y年m月d日 H時i分s秒");
```

Comment:
[MUST]
時間のフォーマットが違うので日時が登録できない状態です！
授業でやったように下記に変更してください 🙆‍♀️ 
```suggestion
$timestamp = date("Y/m/d H:i:s");
```